src/main/java/it/gov/pagopa/pu/workflow/wf/ingestionflow/BaseIngestionFlowFileWFImpl.java

```diff
@@ -111,7 +111,12 @@ protected void mergeErrorDescriptions(IngestionFlowFileResult ingestionResult, S
 
   protected boolean finalizeStatus(Long ingestionFlowFileId, IngestionFlowFileResult result) {
     boolean success = result.getErrorDescription() == null;
-    IngestionFlowFileStatus nextStatus =  success ? IngestionFlowFileStatus.COMPLETED : IngestionFlowFileStatus.ERROR;
+    IngestionFlowFileStatus nextStatus = IngestionFlowFileStatus.COMPLETED;
+    if (!success && result.getProcessedRows() != result.getTotalRows()) {
```

Comment:
@robertopasetti the condition processed != total is limited to an error reading the file.
Actually, the ingestion of DebtPosition after reading the file start the synchronization process, which could bring to errors..
This scenario is still a WARNING or should be an ERROR?